.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
 before_script:
   - gem install sass
   - composer install --prefer-dist
+  - php bin/console assets:install
```

Comment:
should be `php Tests/Integration/skeletion/app/console assets:install` like in the lines below